tests/ref/absolute_img_stacking_a.html

```diff
@@ -0,0 +1,11 @@
+<style>
+body {
+   margin: 0px;
+}
+
+img {
+   position: absolute;
+   margin: 0px;
+}
+</style>
+<img src="rust.png"/>
```

Comment:
Last week, while fixing rendering filtered images, one of the failing tests was a case where an image was absolutely positioned (within div) and as a result, it would create a stacking context and not being rendered at all.